api/urls.py

```diff
@@ -98,7 +98,18 @@ def register_url(self, subclass, viewset):
     url(r'atom/?', views.CreativeWorksAtom(), name='atom'),
     url(r'graph/?', GraphQLView.as_view(graphiql=True)),
     url(r'userinfo/?', ensure_csrf_cookie(views.ShareUserView.as_view()), name='userinfo'),
-    url(r'search/(?!.*_bulk\/?$)(?P<url_bits>.*)', csrf_exempt(views.ElasticSearchView.as_view()), name='search'),
+
+    # only match _count and _search requests
+    url(
+        r'search/(?P<url_bits>(_|[a-zA-Z]+/_)(?:search|count)/{0,1}$)',
```

Comment:
`[a-zA-Z]` can be `\w`. That would match `0-9` and `_` as well, but I think that's okay, if not desirable.